src/_js/leaflet.markersbutton.js

```diff
@@ -0,0 +1,27 @@
+L.MarkersButton = L.Control.extend({
+	options: {
+		position: 'topleft',
+		autoZIndex: true,
+		map: null,
+	},
+	onAdd: function(map) {
+		this._map = map;
+		const container = L.DomUtil.create('div', 'leaflet-control-exiva-button-panel leaflet-bar leaflet-control');
```

Comment:
I'm guessing these references to the `exiva` button are a copy-paste oversight? Let's replace `exiva` with `markers`, here and below.